requirements.dev.in

```diff
@@ -0,0 +1,5 @@
+pylint
+jedi
```

Comment:
I just added it in because it's useful for code completion in the terminal and Emacs. I'm happy to remove it from the dev requirements.